lib/utils/ember.js

```diff
@@ -153,13 +153,19 @@ function isDSModel(node, filePath) {
 }
 
 function isModuleByFilePath(filePath, module) {
-  const fileNameJs = `${module}.js`;
-  const fileNameTs = `${module}.ts`;
-  const folderName = `${module}s`;
+  const expectedFileNameJs = `${module}.js`;
+  const expectedFileNameTs = `${module}.ts`;
+  const expectedFolderName = `${module}s`;
+
+  const filePathParts = filePath.split(path.sep);
+  const actualFolders = filePathParts.slice(0, -1);
+  const actualFileName = filePathParts[filePathParts.length - 1];
```

Comment:
Thoughts on using the `path` methods instead?

```suggestion
  const actualFolders = path.dirname(filePath).split(path.sep);
  const actualFileName = path.basename(filePath);
```

Reply:
Nice, thanks!